kazoo/recipe/lock.py

```diff
@@ -44,9 +44,9 @@ class Lock(object):
     acquired will raise a ``RuntimeError``.
 
     """
-    _NODE_NAME = '__lock__'
```

Comment:
Eh. This also gets used in WLock and RLock, so module scope would be a more appropriate place.

But I'm not sure there's a point in defining symbolic constants for these anyway. They're already just arbitrary tokens, they're readily searchable, and they're unlikely to ever change, so I figure 'explicit' in this case would aid readability.